src/main/java/com/powsybl/openloadflow/network/SimplePiModel.java

```diff
@@ -213,4 +213,9 @@ public PiModel setTapPosition(int tapPosition) {
     public Range<Integer> getTapPositionRange() {
         throw new IllegalStateException(NO_TAP_POSITION_ERROR);
     }
+
```

Comment:
Same remark as for the interface. Can we get rid og this method ?

Reply:
Solved in another comment.